lib/core/presentation/pages/event/event_detail_page/sub_pages/guest_event_application_page/sub_pages/guest_event_application_form_page/widgets/guest_event_application_form_items.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:app/core/application/auth/auth_bloc.dart';
+import 'package:app/core/application/event/event_application_form_bloc/event_application_form_bloc.dart';
+import 'package:app/core/application/event/event_provider_bloc/event_provider_bloc.dart';
+import 'package:app/core/domain/common/common_enums.dart';
+import 'package:app/core/presentation/pages/edit_profile/widgets/edit_profile_field_item.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class GuestEventApplicationFormItems extends StatelessWidget {
+  const GuestEventApplicationFormItems({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final event = context.read<EventProviderBloc>().event;
+    final userProfile = context.read<AuthBloc>().state.maybeWhen(
```

Comment:
getting userProfile data from AuthBloc is not sufficient cause the query in authbloc is only basic fields, should use ProfileBloc